spec.bs

```diff
@@ -228,27 +230,35 @@ The {{SharedStorageWorklet}} object allows developers to supply [=module scripts
   };
 </xmp>
 
-Each {{SharedStorageWorklet}} has an associated boolean <dfn>module added</dfn>, initialized to false.
+Each {{SharedStorageWorklet}} has an associated boolean <dfn>addModule initiated</dfn>, initialized to false.
```

Comment:
nit: doesn't have to be taken care of in this PR but I'd probably make this `<dfn for=SharedStorageWorklet>addModule initiated</dfn>`